src/components/Dashboard/Tiles/Widgets/WidgetsEditModal.tsx

```diff
@@ -135,70 +136,163 @@ const WidgetOptionTypeSwitch: FC<{
   handleChange: (key: string, value: IntegrationOptionsValueType) => void;
 }> = ({ option, widgetId, propName: key, value, handleChange }) => {
   const { t } = useTranslation([`modules/${widgetId}`, 'common']);
-  const { primaryColor } = useColorTheme();
+  const { colorScheme } = useMantineTheme();
+  const info = option.info ?? false;
 
   switch (option.type) {
     case 'switch':
       return (
-        <Switch
-          label={t(`descriptor.settings.${key}.label`)}
-          checked={value as boolean}
-          onChange={(ev) => handleChange(key, ev.currentTarget.checked)}
-        />
+        <Group align="center" spacing="sm">
+          <Switch
+            label={t(`descriptor.settings.${key}.label`)}
+            checked={value as boolean}
+            onChange={(ev) => handleChange(key, ev.currentTarget.checked)}
+            {...option.inputProps}
+          />
+          {info &&
+            <Tooltip.Floating
+              label={t(`descriptor.settings.${key}.info`)}
+              position="right-start"
+              c={ colorScheme === 'light' ? "black" : "dark.0" }
+              color={ colorScheme === 'light' ? "dark.0" : "dark.6" }
+              styles={{ tooltip: { maxWidth: 300, }, }}
+              multiline
```

Comment:
Instead of repeating the tooltip, you could put it into a own component and use that, so you don't need to modify all when something changes.